tests/plugins/tableselection/tableselection.js

```diff
@@ -20,7 +20,7 @@
 	}
 
 	var tests = {
-		// (#258, #tp2247)
+		// (http://dev.ckeditor.com/ticket/258, #tp2247)
```

Comment:
Since this issue is not available to the public, I decided to remove it all along.